messenger_users/views.py

```diff
@@ -79,6 +80,35 @@ def confirm_linkedin_profile(request):
 # def browse_saved_jobs(request):
 #     pass
 
+def search_linkedIn_profiles(request):
+    data = json.loads(request.body)
+    messenger_id = data['messenger_id']
+    username = data['linkedin_username']
+
+    searchObject = Search()
+    
+    urlArray = searchObject.get_user_profile_suggestions(username)
+    
+    gallery_message = GalleryMessage("square")
```

Comment:
user ButtonMessage instead of GalleryMessage, this is because we don't have a picture to send over. So now instead of having one GalleryMessage containing ~5 buttons in it. We'll have 5 ButtonMessage, each with one button and message in it. In this case the button will have the message of "It's me!", but the URL does not lead to addr (linkedin_address). it should rather be  a block button instead of URLbutton, and lead to the 'VerifyLinkedIn' block.

Reply:
BlockButton because after the user clicked 'this is me', the next step for us is to scrape the linkedin profile, which happens in verifyLinkedin